cr-examples/eventstreams/01-light-insecure.yaml

```
@@ -15,7 +15,7 @@ kind: EventStreams
 metadata:
   name: light-insecure
 spec:
-  version: 11.0.4
+  version: 11.1.0
```

The one-to-one management change means the only valid value for the ES CR spec.version field is now `latest`
```suggestion
  version: latest
```